INSTALL.md

```diff
@@ -0,0 +1,328 @@
+# Basic installation instructions
+
+Stable version installation on Unix-like systems (e.g. GNU/Linux or *BSD):
+* Check your OS repository for already built packages.
+* Check https://celestia.space/download.html.
+* Check https://bintray.com/celestia if it contains packages for your system.
+
+Stable version installation on Windows and OSX:
```

Comment:
OSX should be replaced with macOS

Reply:
Sure, althought it isn't in the text-only version either.